### PR TITLE
Add function pointer parse.

### PR DESCRIPTION
**Pull request by Erignik**
Please review and merge.

### CODE REVIEW

File: CppHeaderParser/CppHeaderParser.py [hunk 1]
```diff
@@ -2375,13 +2375,30 @@ def _evaluate_method_stack(self):
         self.stack = []
         self.stmtTokens = []
 
+    def _function_point_typedef_parse(self, stack):
+        idx = stack.index("typedef")
+        expression = "".join(stack[idx + 1 :])
+        pattern = re.compile(r".*?\(.*?\*(.*?)\).*?\(.*?\).*?")
```
Comment (virtuald) [inline CppHeaderParser/CppHeaderParser.py:2381]: Can you lift this out to be a global so it's not computed over and over again?

**Review by virtuald — COMMENTED** [review]
LGTM other than the minor comment. Thanks!

File: CppHeaderParser/CppHeaderParser.py (hunk 1 above)
Comment (Erignik) [reply CppHeaderParser/CppHeaderParser.py]: yeah, it's a good suggestion and already done.

**Comment by virtuald** [comment]
Thanks!

**Pull request merged**